### PR TITLE
add a note about raft-engine

### PR DESCRIPTION
**Pull request by tabokie**
Signed-off-by: tabokie <xy.tao@outlook.com>

### What is changed, added or deleted? (Required)

Synchronized in https://github.com/pingcap/docs/pull/7512

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7512
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1351,6 +1351,8 @@ Raft Engine 相关的配置项。
 
 > **注意：**
 >
+> 初次打开 Raft Engine，需要进行数据格式切换，这一过程可能花费数十秒。
+>
 > TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），并重启 TiKV 使配置生效，否则会导致集群降级后无法正常开启。
```
Comment (en-jin19) [inline tikv-configuration-file.md:1356]:
```suggestion
> - 初次开启 Raft Engine 时，TiKV 会将 RocksDB 的数据格式切换至 Raft Engine 的数据格式。因此，初次开启时，你需要数十秒。
> - TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），并重启 TiKV 使配置生效，否则会导致集群降级后无法正常开启。
```

**Comment by en-jin19** [comment]
@tabokie Could you please invite a tech reviewer to review this PR? thank you!

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1351,7 +1351,8 @@ Raft Engine 相关的配置项。
 
 > **注意：**
 >
-> TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），并重启 TiKV 使配置生效，否则会导致集群降级后无法正常开启。
+> - 初次开启 Raft Engine 时，由于 TiKV 会将 RocksDB 的数据格式切换至 Raft Engine 的数据格式，你需要等待数十秒。
```
Comment (en-jin19) [inline tikv-configuration-file.md:1354]:
```suggestion
> - 初次开启 Raft Engine 时，TiKV 会把 RocksDB 的数据格式切换至 Raft Engine 的数据格式，因此，你需要等待数十秒。
```

Comment (en-jin19) [reply tikv-configuration-file.md:1354]: @tabokie PTAL, thanks!

**Review by en-jin19 — APPROVED** [review]
LGTM

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1351,7 +1351,8 @@ Raft Engine 相关的配置项。
 
 > **注意：**
 >
-> TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），并重启 TiKV 使配置生效，否则会导致集群降级后无法正常开启。
+> - 第一次开启 Raft Engine 需要等待几十秒，TiKV 会在此过程中将原有的 RocksDB 数据转移至 Raft Engine 中。
```
Comment (Connor1996) [inline tikv-configuration-file.md:1354]:
```suggestion
> - 第一次开启 Raft Engine ，TiKV 启动时间会较长，需要额外几十秒，会在此过程中将原有的 RocksDB 数据转移至 Raft Engine 中。
```

**Review by Connor1996 — APPROVED** [review]
LGTM

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1351,7 +1351,8 @@ Raft Engine 相关的配置项。
 
 > **注意：**
 >
-> TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），并重启 TiKV 使配置生效，否则会导致集群降级后无法正常开启。
+> - 第一次开启 Raft Engine ，TiKV 启动时间较长，需要额外几十秒，会在此过程中将原有的 RocksDB 数据转移至 Raft Engine 中。
+> - TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），并重启 TiKV 使配置生效，否则会导致集群降级后无法正常开启。
```
Comment (tabokie) [inline tikv-configuration-file.md:1355]:
```suggestion
> - TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`，并重启 TiKV 使配置生效），否则会导致集群降级后无法正常开启。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1351,7 +1351,8 @@ Raft Engine 相关的配置项。
 
 > **注意：**
 >
-> TiDB v5.4.0 版本的 Raft Engine 数据格式与之前版本不兼容。因此，当要将 TiDB 集群降级至 v5.4.0 以前的版本时，你需要在降级**之前**先关闭 Raft Engine（即把 `enable` 配置项设置为 `false`），并重启 TiKV 使配置生效，否则会导致集群降级后无法正常开启。
+> - 第一次开启 Raft Engine ，TiKV 启动时间较长，需要额外几十秒，会在此过程中将原有的 RocksDB 数据转移至 Raft Engine 中。
```
Comment (en-jin19) [inline tikv-configuration-file.md:1354]:
```suggestion
> - 第一次开启 Raft Engine 时，TiKV 会将原有的 RocksDB 数据转移至 Raft Engine 中。因此，TiKV 的启动时间会比较长，你需要额外等待几十秒。
```

**Pull request merged**